bootstrap/sql/migrations/flyway/com.mysql.cj.jdbc.Driver/v016__create_db_connection_info.sql

```diff
@@ -0,0 +1,7 @@
+-- Update BigQuery model for gcpCredentials to move `gcpCredentialsPath` value to `gcpCredentialsPath.path`
+UPDATE dbservice_entity
+SET json = JSON_INSERT(
+    JSON_REMOVE(json, '$.connection.config.credentials.gcpCredentialsPath'),
```

Comment:
@ayush-shah if you can help me on this, I don't know how to write migration haha.

Reply:
No worries @aballiet, I will take a look in couple of hours. Thanks, I will try to have this merge by tomorrow, Thanks.🙏